src/System.Management.Automation/engine/CommonCommandParameters.cs

```diff
@@ -78,6 +78,7 @@ public SwitchParameter Debug
         /// This parameter tells the command what to do when an error occurs.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By CustomPipeName.Can enter, exit, and re-enter using CustomPipeName
```powershell
Expected $true, because The script was able to re-enter another process and grab the pipe of 'tmizsa3o.tj3'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 208
208:                     Should -BeTrue -Because "The script was able to re-enter another process and grab the pipe of '$pipeName'."
```